lib/generators/pageflow/theme/theme_generator.rb

```diff
@@ -13,9 +13,10 @@ def copy_template
         directory('images/pageflow/themes/default', themes_path('images', name))
 
         template('stylesheets/pageflow/themes/default.css.scss', themes_path('stylesheets', "#{name}.css.scss")) do |content|
-          content.gsub!('$theme-name: "default";', %Q'$theme-name: "#{name}";')
           content.gsub!('@import "./default/', %Q'@import "./#{name}/')
         end
+
+        gsub_file themes_path('stylesheets', "#{name}/variables.css.scss"), /default/, name
```

Comment:
Replacing all occurrences of the string `default` might be dangerous if there ever is a variable like `pageflow-default-color`, right? Maybe we should stick with matching `$theme-name: ...`?